actors/market/src/ext.rs

```diff
@@ -79,6 +79,7 @@ pub mod verifreg {
 }
 
 pub mod datacap {
+    pub const BALANCE_OF_METHOD: u64 = 13;
     pub const TRANSFER_FROM_METHOD: u64 = 15;
 }
```

Comment:
FYI @arajasek  we need to catch this when rebasing integration/builtin-api

Reply:
Anything remaining to do here?